src/subgraph/actions/getMostRecentSubdomainsById.ts

```diff
@@ -6,44 +6,50 @@ import { DomainsQueryDto } from "../types";
 import { convertDomainDtoToDomain, performQuery } from "./helpers";
 
 const logger = getLogger().withTag("subgraph:actions:getRecentSubdomainsById");
+const MAX_RECORDS = 5000;
 
-export const getRecentSubdomainsById = async <T>(
+export const getMostRecentSubdomainsById = async <T>(
   apolloClient: ApolloClient<T>,
-  domainId: string
+  domainId: string,
+  count = 1000
```

Comment:
Consider letting the caller pass the `skip` starting value, then default to 0 like you are if not provided